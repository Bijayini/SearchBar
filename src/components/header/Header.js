import React from 'react';

import './header.scss';

const Header = () => (
  <header>
    <a href="#" className="brand">
      User Search
    </a>
  </header>
);

export default Header;
