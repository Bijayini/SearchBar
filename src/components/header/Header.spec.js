import React from "react";
import { render } from "@testing-library/react";
import Header from "./Header";

describe("Header component", () => {
  it("should render Header", () => {
    const {baseElement} = render(<Header/>);

    expect(baseElement).toMatchSnapshot();
  });
});
