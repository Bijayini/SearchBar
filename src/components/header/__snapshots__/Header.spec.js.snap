// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render Header 1`] = `
<body>
  <div>
    <header>
      <a
        class="brand"
        href="#"
      >
        User Search
      </a>
    </header>
  </div>
</body>
`;
